docker/Dockerfile

```diff
@@ -7,7 +7,15 @@ SHELL ["/bin/bash", "-euxo", "pipefail", "-c"]
 
 RUN \
   apt-get update; \
-  apt-get install --no-install-recommends -y git hub openssh-client
+  apt-get install --no-install-recommends -y git hub openssh-client \
+
+#Install GitHub Cli
+Run \
+  curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg \
+  && chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg \
```

Comment:
`chmod` is unlikely to be required; you're running all commands as **root** anyway